tests/vm_test.go

```diff
@@ -648,7 +648,7 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			})
 
 			Context("Using RunStrategyRerunOnFailure", func() {
-				It("should stop a running VM", func() {
+				It("[test_id:2186] should stop a running VM", func() {
```

Comment:
I am missing the action of restart the VM and stop VM and verify vm's runStrategy is now Halted verify vm.status.stateChangeRequests is not present in vm's manifest

Reply:
Confirmed test is correct in offline conversation with @ILpinto